src/components/Atoms/Timestamp/Timestamp.tsx

```diff
@@ -0,0 +1,25 @@
+import { timestampParser } from "@/utils/functions";
+import { type TimestampProps } from "@/utils/types/atoms.types";
+import { useState } from "react";
+import { ClockIcon } from "@radix-ui/react-icons";
+
+export const Timestamp: React.FC<TimestampProps> = ({
+    timestamp,
+    defaultType = "descriptive",
+}) => {
+    const [relativeTime, setRelativeTime] = useState<boolean>(
+        defaultType === "relative"
+    );
+
+    return (
+        <span className="inline-flex items-center gap-x-1 text-black dark:text-slate-50">
```

Comment:
Change font color to theme colors

Reply:
Resolved in : 6242dbcd261df6ec27ff52fd16b1936ece98cd5d